app/Components/header/menu/Menu.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import s from '../headerStyle.module.css';
+
+const data = [
+    {
+        title: "Main",
+        href: '#Main'
+    },
+    {
+        title: "Invitation",
+        href: '#Invitation'
+
+    },
+    {
+        title: "Schedule",
+        href: '#Schedule'
+    },
+    {
+        title: "Contacts",
+        href: '#Contacts'
+    }
+]
```

Comment:
Move to consts to avoid the presence of duplicate values.

Reply:
this component is not used, I will remove it